tapcfg/server.go

```diff
@@ -156,11 +156,36 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 		assetStore, proofFileStore,
 	)
 
+	// If no default proof courier address is set, use the fallback hashmail
+	// address.
+	fallbackHashmailCourierAddr := fmt.Sprintf(
+		"%s://%s", proof.ApertureCourier, fallbackHashMailAddr,
+	)
+	proofCourierAddr, err := proof.ParseCourierAddr(
+		fallbackHashmailCourierAddr,
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse fallback proof "+
+			"courier address: %v", err)
+	}
+
+	// If default proof courier address is set, use it as the default.
+	if cfg.DefaultProofCourierAddr != "" {
+		proofCourierAddr, err = proof.ParseCourierAddr(
```

Comment:
I think would also make sense to eventually enable this to be configurable when one goes to create an address. Similarly, I think it would eventually be useful to support multiple couriers encoded in a given addr. 

I think non-blocking now as we don't need it, but can imagine an itest in the future where two are specified, one is down, but the other can be used for the final transfer.

Reply:
> enable [`DefaultProofCourierAddr`] to be configurable when one goes to create an address

Perhaps via a new RPC argument flag for setting the given address as the new default proof courier address. I'll open an issue for that.